Example/ios/ReanimatedExample/main.m

```diff
@@ -5,6 +5,6 @@
 int main(int argc, char *argv[])
 {
   @autoreleasepool {
-    return UIApplicationMain(argc, argv, nil, NSStringFromClass([AppDelegate class]));
+t -    return UIApplicationMain(argc, argv, nil, NSStringFromClass([AppDelegate class]));
```

Comment:
not

Reply:
![image](https://github.com/software-mansion/react-native-reanimated/assets/40713406/2607eae5-c3f4-4174-930d-bed9f83f5485)